### DISCUSSION TITLE
Is it possible to load a set of features at initialization step ?

### DISCUSSION THREAD

**Discussion by ghost** [Flagsmith/flagsmith#1893] [Q&A] [answered]
Hi,
I'm looking for ways to load some features just after the docker image bootstraps.
It is possible to wait for it, and from somewhere else use [Direct API Access](https://docs.flagsmith.com/clients/rest) to load them.

But is there a native **Flagsmith** way of doing it part of Docker deployment ?
Thanks a lot .

gz#378

**Comment by dabeeeenster** [comment, marked as answer]
Interesting question! There are two options here Direct API access as you mention, but also potentially via the [Terraform Provider](https://docs.flagsmith.com/integrations/terraform). You would need to do some work via the API regardless to creaet the Organisation, Project etc. 

We don't currently provide any help in this regard!

**Reply by ghost** [reply]
Thanks @dabeeeenster,
Terraform is not part of our stack at this moment.
Otherwise, I tried to play around docker `ENTRYPOINT` and `CMD` part of **Docker Compose**. But this is always catchy and could interfere in a wrong way with **Flagsmith**.
~I will continue to try to load them from somewhere else using Direct API.~ (actually from the client app itself, like the python code example) 
Thanks a lot !
